File: Java/src/Prime_Number.java
/**
 * Prime Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-003
 */
import java.util.*;
public class Prime_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Prime Number Checker\n~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        boolean flag=false;
        for(int i=2;i<=n/2;i++)
        {
            if(n%i==0)
            {
                flag=true;
                break;
            }
        }
        if(!flag)
            System.out.print(n+" is a Prime number.");
        else
            System.out.print(n+" is not a Prime number.");        
    }
}

File: Python/Prime_Number.py
print("Prime Number Checker\n")
n = int(input("Enter the number : "))
flag = True
for i in range(2, n//2+1) :
	if n%i == 0:
		flag = False
		break
if(flag) and n!=1 :
	print(f"{n} is a prime number.")
else :
	print(f"{n} is not a prime number.")